### DISCUSSION TITLE
Is there a way to add methods to the existing livewire page through the plug-in boot and not trait or inheritance?

### DISCUSSION THREAD

**Discussion by killjin** [filamentphp/filament#7402] [Help]
Is there a way to add methods to the existing livewire page through the plug-in boot and not trait or inheritance?

Don't know the name of an existing page, how do I do it? For example [List|Manage]Users.php

Or is there a way to execute Table's Macroable method with page JS?

Or JS: this.$wire.$model.saveXXX()

**Comment by pboivin** [comment]
Seems a bit hacky, no? What problem are you trying to solve?
